src/components/artwork_filter/headline.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react"
+
+import { secondary } from "../../assets/fonts"
+import labelMap from "./param_map"
+
+import { compact } from "lodash"
+import styled from "styled-components"
+
+interface HeadlineProps extends React.HTMLProps<Headline> {
+  medium: any,
+  price_range: any,
+  dimension_range: any,
+}
+
+export class Headline extends React.Component<HeadlineProps, null> {
+  size() {
+    if (this.props.dimension_range && this.props.dimension_range !== "*") {
+      return this.props.dimension_range
+    }
+    return false
+  }
+
+  medium() {
+    if (this.props.medium && this.props.medium !== "*") {
+      return this.props.medium
+    }
+    return "Works"
+  }
+
+  priceRange() {
+    if (this.props.price_range && this.props.price_range !== "*") {
+      return this.props.price_range
+    }
+    return false
```

Comment:
Why not 
```javascript
const { price_range } = this.props

if (price_range && price_range !== "*") {
  return price_range
}
```

I always think to myself if `this.props` is used more than once its best to destructure a value out, especially because you might grab more props later.